bin/console

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "solidus"
+
+module IRBSolidusHelpers
+  def dummy_app
+    @dummy_app ||= begin
+      print "Loading the dummy app..."
+      ENV['DATABASE_URL'] ||= 'sqlite3::memory:?pool=1'
+
+      require "spree/testing_support/dummy_app"
+
+      class << (Rails.application)
+        # Allow to run migrations from the console
+        def migrate!
+          ActiveRecord::Migrator.migrations_paths =
+            Rails.application.migration_railties.flat_map do |engine|
+              (engine.paths['db/migrate'] if engine.respond_to?(:paths)).to_a
+            end
+          ActiveRecord::Migration.verbose = false
+          ActiveRecord::Base.connection.migration_context.run(:up, '20160101010000_solidus_one_four'.to_i)
```

Comment:
Why this up?

Reply:
Without running that initial migration in isolation it fails with this error:

```
SQLite3::SQLException: no such table: spree_users
```